src/include/parser/simple/attach_database.h

```diff
@@ -1,6 +1,6 @@
 #pragma once
 
-#include "parsed_data/attach_info.h"
```

Comment:
I wouldn't create a simple folder for parser. There is no need to unify these statement at parser level.